crates/bevy_audio/src/sinks.rs

```diff
@@ -0,0 +1,214 @@
+use bevy_math::Vec3;
+use bevy_reflect::TypeUuid;
+use bevy_transform::prelude::Transform;
+use rodio::{Sink, SpatialSink};
+
+/// Common interactions with an audio sink.
+pub trait AudioSinkExt {
```

Comment:
Not a huge fan of the `Ext` naming convention, especially in this case. I'd prefer something more descriptive like `AudioSinkPlayback` or a rename of the "non spatial sink" to something else which would allow us to call this trait `AudioSink`.

Or alternatively, do we need this trait at all? Can we just do separate impls across both sink types? Idk if we really need to provide the ability to abstract over both, and it removes one more trait / name that people need to import / think about.

Reply:
I like the trait to keep the api and the documentation common. I renamed it